viadot/tasks/mediatool.py

```diff
@@ -0,0 +1,196 @@
+from typing import Any, Dict, List, Literal
+import pandas as pd
+
+from prefect import Task
+from prefect.utilities import logging
+from prefect.utilities.tasks import defaults_from_attrs
+
+from viadot.sources import Mediatool
+from ..exceptions import CredentialError
+
+from viadot.task_utils import *
```

Comment:
is there a option to avoid using *?

Reply:
I had a problem with circular import (`..import credentials_loader`). This line fixed this problem for now.